WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.java

```diff
@@ -0,0 +1,178 @@
+package org.wordpress.android.ui.mysite;
```

Comment:
I just did `Analyze > Inspect code` on this fragment and it found zero problems. Nice work!